sql-statements/sql-statement-admin-show-ddl.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ADMIN SHOW DDL [JOBS|QUERIES] | TiDB SQL Statement Reference
+summary: An overview of the usage of ADMIN for the TiDB database.
+aliases: ['/docs/dev/sql-statements/sql-statement-admin/','/docs/dev/reference/sql/statements/admin/']
```

Comment:
We can't put the same aliases in two files (`sql-statement-admin-show-ddl.md` and `sql-statement-admin.md`) at the same time. This might cause an error for the original links.

Reply:
I've removed it, this was an error.